bot_api_py/events/connected_event.py

```diff
@@ -0,0 +1,5 @@
+from bot_api_py.events.connection_event import ConnectionEvent
+
+class ConnectedEvent(ConnectionEvent):
```

Comment:
TODO: Add ConnectedEvent(URI serverUri) constructor

Reply:
Since `ConnectedEvent`'s ctor is exactly the same as `ConnectionEvent`, I don't believe it is needed. It'll simply inherit `ConnectionEvent`'s ctor.

See [this colab](https://colab.research.google.com/drive/1RlQJSf6e1w0UeUWGY9e4kFhC6gbXUmnw?usp=sharing) for a simple example.